README.md

```diff
@@ -58,6 +58,22 @@ module.exports = function (environment) {
 };
 ```
 
+#### `page-title` Service
+
+If you want to be notified when the page title has been updated, you can extend and override the `page-title` service and provide your own `titleDidUpdate` hook. The `titleDidUpdate` hook receives the new title as its sole argument.
+
+```javascript
+// app/services/page-title.js
+
+import EmberPageTitleService from 'ember-page-title/services/page-title';
+
+class PageTitleService extends EmberPageTitleService {
+  titleDidUpdate(/* newTitle */) {
```

Comment:
```suggestion
  titleDidUpdate(title) {
```

Since it's the only reason someone would extend the service atm I think we can assume they want the title. And the hook explains the argument, so no need for `new` in `newTitle`.

Reply:
Plus the comment below 😄

Reply:
Should be okay now.